util/src/builder/mod.rs

```diff
@@ -1,3 +1,6 @@
 //! Builders for large structs.
 
+mod callback_data;
 pub mod command;
+
+pub use callback_data::CallbackDataBuilder;
```

Comment:
can we use `self::` here: 
```suggestion
pub use self::callback_data::CallbackDataBuilder;
```